public/pages/Indices/components/CloseIndexModal/CloseIndexModal.tsx

```diff
@@ -36,13 +39,21 @@ export default function CloseIndexModal(props: CloseIndexModalProps) {
     return null;
   }
 
+  const showWarning = selectedItems.filter((item) => filterByMinimatch(item as string, SYSTEM_INDEX)).length > 0;
+
   return (
     <EuiModal onClose={onClose}>
       <EuiModalHeader>
         <EuiModalHeaderTitle>Close indices</EuiModalHeaderTitle>
       </EuiModalHeader>
 
       <EuiModalBody>
+        <>
+          <EuiCallOut color="warning" hidden={!showWarning}>
+            You are closing system-like index, please be careful before you do any change to it.
```

Comment:
Please change system-like to system index.

Reply:
OK, I'll change it.